openapi/publicapi_spec.json

```diff
@@ -1558,6 +1558,32 @@
           }
         }
       },
+      "CardDetailsFromResponse" : {
```

Comment:
Why has this been added in this PR?

Reply:
I have no idea. It occurred when I updated the openapi spec. I assume there was a previous change that wasn't updated correctly